v0/sources/iterable/transform.js

```diff
@@ -0,0 +1,47 @@
+const path = require("path");
+const fs = require("fs");
+const md5 = require("md5");
+const Message = require("../message");
+
+// import mapping json using JSON.parse to preserve object key order
+const mapping = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, "./mapping.json"), "utf-8")
+);
+
+function process(event) {
+  // throw an error if (email, eventName) are not present
+  if (!(event.email && event.eventName)) {
```

Comment:
Shouldn't we check email property inside data field it can be present inside data field but not in root
@ItsSudip ?

Reply:
According to their [documentation](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#webhook-request-body) email will be there always.

Reply:
Got it